oak_functions/loader/src/main.rs

```diff
@@ -168,6 +171,13 @@ async fn async_main(opt: Opt, config: Config, logger: Logger) -> anyhow::Result<
     let address = SocketAddr::from((Ipv6Addr::UNSPECIFIED, opt.http_listen_port));
     let tee_certificate = vec![];
 
+    let aggregator = match &config.metrics {
+        Some(metrics_config) => Some(Arc::new(Mutex::new(PrivateMetricsAggregator::new(
+            metrics_config,
+        )?))),
+        None => None,
+    };
```

Comment:
(optional) could use `config.metrics.map(|c| ... )`

Reply:
I previously used that before `PrivateMetricsAggregator::new` returned `Result<...>`, but the `?` means that it can no longer be used in a closure.